sktime/forecasting/base/_base.py

```diff
@@ -34,7 +34,7 @@ class name: BaseForecaster
 
 __author__ = ["mloning", "big-o", "fkiraly", "sveameyer13", "miraep8", "ciaran-g"]
 
-__all__ = ["BaseForecaster"]
+__all__ = ["BaseForecaster", "BaseGlobalForecaster"]
```

Comment:
the idea is later a merge into the base class, right? So we should make this private for now, start with an underscore

Reply:
renamed to `_BaseGlobalForecaster`